workflow/controller/workflowpod.go

```diff
@@ -321,6 +319,32 @@ func (woc *wfOperationCtx) createWorkflowPod(ctx context.Context, nodeName strin
 		{Name: common.EnvVarDeadline, Value: woc.getDeadline(opts).Format(time.RFC3339)},
 	}
 
+	if woc.controller.Config.Progress.Enabled == nil || *woc.controller.Config.Progress.Enabled {
+		// setup defaults for ticks if not set or set to 0 value.
+		patchPodTickDuration := woc.controller.Config.Progress.PatchPodAnnotationTickDuration
+		if patchPodTickDuration == nil {
+			patchPodTickDuration = &metav1.Duration{Duration: 1 * time.Minute}
```

Comment:
could you add ` woc.controller.Config.Progress.GetPatchPodAnnotationTickDuration()`